src/thunks/getActiveTickets/getActiveTickets.test.js

```diff
@@ -0,0 +1,57 @@
+import { getActiveTickets } from './';
+import * as actions from '../../actions';
+
+describe('getActiveTickets', () => {
+  let url, mockDispatch, thunk, mockTickets;
+
+  beforeEach(() => {
+    url = 'https://fathomless-shore-89603.herokuapp.com/api/v1/tickets';
+    mockDispatch = jest.fn();
+    thunk = getActiveTickets();
+    mockTickets = ['ticket1', 'ticket2']
```

Comment:
We can dry up or code by putting our mock Data in our mock folder as the application becomes more robust.